app/helpers/application_helper/button/chargeback_report_download.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationHelper::Button::ChargebackReportDownload < ApplicationHelper::Button::Basic
+    def role_allows_feature?
+      role_allows?(:feature => 'chargeback_reports')
+    end
```

Comment:
I think you don't need this method and the custom button class if you set the feature in the button specification below.

Reply:
looking into this bit further, looks like this code is dead